install/running_install.adoc

```diff
@@ -593,117 +607,19 @@ of your etcd hosts in the following:
 [[verifying-multiple-masters-haproxy]]
 ==== Verifying Multiple Masters Using HAProxy
 
-If you installed multiple masters using HAProxy as a load balancer, browse to
-the following URL according to your *[lb]* section definition and check
-HAProxy's status:
+If you installed multiple masters using HAProxy as a load balancer, open
+the following URL and check HAProxy's status:
 
 ----
-http://<lb_hostname>:9000
+http://<lb_hostname>:9000 <1> 
 ----
+<1> Provide the load balancer host name listed in the `[lb]` section of your
+inventory file.
 
 You can verify your installation by consulting the
 https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Load_Balancer_Administration/ch-haproxy-setup-VSA.html[HAProxy
 Configuration documentation].
 
-[[optionally-securing-builds]]
-== Optionally Securing Builds
```

Comment:
I don't think this subsection made sense, so I moved it to a prereq.

Reply:
You're right. The section makes it sound like you do it with installation. Thank you!